executor/adapter.go

```diff
@@ -175,6 +178,7 @@ type ExecStmt struct {
 	isPreparedStmt    bool
 	isSelectForUpdate bool
 	retryCount        uint
+	IsInMultiQueries  bool
```

Comment:
We do not use InMultiQueries after this refactor https://github.com/pingcap/tidb/pull/16056